tokens/src/core/components/Card.json

```diff
@@ -15,7 +15,7 @@
       },
       "columns": {
         "margin": { "source": "$card-columns-margin", "$value": "{spacing.card.spacer.y}" },
-        "count": { "source": "$card-columns-count", "$value": "3" },
+        "count": { "source": "$card-columns-count", "$value": "3", "$type": "number" },
```

Comment:
Bug fix around `$type` for a _number_ value, as this token does not define a dimension type currently defined above.